draft-ietf-masque-connect-udp-listen.md

```diff
@@ -153,11 +170,103 @@ UDP Payload:
 : The unmodified UDP Payload of this proxied UDP packet (referred to as "data
 octets" in {{UDP}}).
 
+# Context ID {#contextid}
+
+Context ID values MUST always be greater than 0. A payload with value 0 MUST
+be dropped by the receiving party.
+
+An even context ID MUST be used by clients to send packets to the proxy, and an odd
+context ID MUST be used by the proxy to send packets to the client. A Client and
+a proxy MUST exhange COMPRESSION_ASSIGN packets for a given context ID to guarantee
+that all payloads are received and processed by the other party, see
+{{compression}}. They MAY pre-emptively use Context IDs not acknowledged by the
```

Comment:
This sentence discusses acknowledging context IDs, but that concept hadn't been introduced yet

Reply:
The preceeding paragraph cites context ID from section 4 CONNECT UDP. Should I add a one liner to explain it a bit more?

Reply:
But connect-udp doesn't define the concept of acknowledging context IDs. So I'd either define it quickly or add a forward reference

Reply:
Added the reference to Context IDs from the original RFC